### PR TITLE
PMM-4534 Added cli options for TLS in PostgreSQL

### PR DESCRIPTION
**Pull request by percona-csalguero**
- Added tls param to enable TLS support
- Added tls-skip-verifify so set the ssl-mode.
  - true : use ssl-mode = required
  - false: use ssl-mode = verify-full

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/pmm-admin?pullRequest=56) <br/>All committers have signed the CLA.

**Review by BupycHuk — APPROVED** [review]

**Review by AlekSi — CHANGES_REQUESTED** [review]
Please also add it to `pmm-admin add postgresql` Management API command.

File: commands/inventory/add_service_postgresql.go [hunk 1]
```diff
@@ -100,4 +104,7 @@ func init() {
 	AddServicePostgreSQLC.Flag("cluster", "Cluster name").StringVar(&AddServicePostgreSQL.Cluster)
 	AddServicePostgreSQLC.Flag("replication-set", "Replication set name").StringVar(&AddServicePostgreSQL.ReplicationSet)
 	AddServicePostgreSQLC.Flag("custom-labels", "Custom user-assigned labels").StringVar(&AddServicePostgreSQL.CustomLabels)
+	AddServicePostgreSQLC.Flag("tls", "Enable TLS when connecting to PostgreSQL").BoolVar(&AddServicePostgreSQL.TLS)
+	AddServicePostgreSQLC.Flag("tls-skip-verify", "Skip TLS certificates validation (use ssl-mode=required instead of verify-full").
+		BoolVar(&AddServicePostgreSQL.TLSSkipVerify)
```
Comment (AlekSi) [inline commands/inventory/add_service_postgresql.go]: There is no TLS options for service. I will remove them.

File: commands/management/add_postgresql.go [hunk 2]
```diff
@@ -183,6 +187,9 @@ func init() {
 	AddPostgreSQLC.Flag("custom-labels", "Custom user-assigned labels").StringVar(&AddPostgreSQL.CustomLabels)
 
 	AddPostgreSQLC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddPostgreSQL.SkipConnectionCheck)
+	AddPostgreSQLC.Flag("tls", "Enable TLS for PostgreSQL connection").BoolVar(&AddPostgreSQL.TLS)
+	AddPostgreSQLC.Flag("tls-skip-verify", "Skip TLS certificate validation (use ssl-mode= require instead of verify-full").
+		BoolVar(&AddPostgreSQL.SkipConnectionCheck)
```
Comment (AlekSi) [inline commands/management/add_postgresql.go]: Wrong variable. I will update.

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Will merge tomorrow morning.

**Pull request merged**